doc/news/changes/minor/20170702AndreasKergassnerJean-PaulPelteret-1b

```diff
@@ -0,0 +1,4 @@
+Fixed: FullMatrix::determinant() is now implemented for square matrices
+of size greater than 3.
```

Comment:
please mention in function documentation that for n>3 determinant is computed via LU using Lapack.